javascript-source/core/commandCoolDown.js

```diff
@@ -31,19 +31,41 @@
 
     $.raffleCommand = null;
 
+    const   Type = {
+                User: 'user',
+                Global: 'global',
+            },
+            Operation = {
+                UnSet: -1,
+                UnChanged: 0,
+            };
+
     /*
      * @class Cooldown
-     *
      * @param {String}  command
-     * @param {Number}  seconds
-     * @param {Boolean} isGlobal
+     * @param {Number}  globalSec
+     * @param {Number}  userSec
      */
-    function Cooldown(command, seconds, isGlobal) {
-        this.isGlobal = isGlobal;
-        this.command = command;
-        this.seconds = seconds;
-        this.cooldowns = [];
-        this.time = 0;
+    function Cooldown(command, globalSec, userSec) {
+            this.command = command;
+            this.globalSec = globalSec;
+            this.userSec = userSec;
+            this.userTimes = [];
+            this.globalTime = 0; 
+    }
+
+    /*
+     * @function Stringify
+     * @param {String}  command
+     * 
+     * @return {JSON String}
+     */ 
+    function toJSONString(command) {
+        return JSON.stringify({
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
    return JSON.stringify({
```